src/components/editor/EditorAppController.jsx

```diff
@@ -15,6 +29,16 @@ export default class EditorAppController extends BaseComponent {
     super(props);
     this.handleDisableLink = this.handleDisableLink.bind(this);
     this.resetGhostInfo = this.resetGhostInfo.bind(this);
+    this.state = {
+      isLoggedIn: true, // TODO: change to false when log in works
+      isRestartedSnackBarOpen: false,
+      isGhostResetSnackBarOpen: false,
+    };
+  }
+
+  componentWillMount() {
+    // Used for material-ui magic;
+    injectTapEventPlugin(); // Will be removed when React supports onTouchTap() soon
```

Comment:
It's for faster rendering of a click. It's required for the visuals of Material UI to work properly